docs/api.rst

```diff
@@ -17,7 +17,7 @@ Servers
 Clients
 =======
 
-Since it is an HTTP API and the records are public, you can simply use any HTTP client, like `curl <http://curl.haxx.se>`_ or `HTTPie <https://httpie.org>`_.
+Since it is an HTTP API and the records are public, you can simply use any HTTP client, like `curl <http://curl.haxx.se>`_ or `HTTPie <https://httpie.org>`__.
```

Comment:
Without it you get an warning from `sphinx` and with the double underscore the warning goes away. 
You get this:
```
/app/docs/api.rst:4: WARNING: Duplicate explicit target name: "httpie".
```
See https://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text/14067756#14067756

It's because there are two hyperlinks with the same text but different URL. 

The two different urls were `https://httpie.org` and `https://httpie.org/`. Subtle. 
As a matter of fact, if I correct that so both URLs are `https://httpie.org` instead, then I can use a single underscore instead.